operator/src/main/java/com/github/streamshub/console/api/v1alpha1/spec/ConsoleSpec.java

```diff
@@ -0,0 +1,55 @@
+package com.github.streamshub.console.api.v1alpha1.spec;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.sundr.builder.annotations.Buildable;
+
+@Buildable(builderPackage = "io.fabric8.kubernetes.api.builder")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ConsoleSpec {
+
+    String hostname;
+
+    Images images = new Images();
+
+    List<KafkaCluster> kafkaClusters = new ArrayList<>();
+
+    // TODO: copy EnvVar into console's API to avoid unexpected changes
+    List<EnvVar> env = new ArrayList<>();
```

Comment:
I left this TODO here because I thought it might be bad form to embed something from fabric8 directly in the CRD, but I see there is precedent for that in the Strimzi API as well. Unless anyone has reservations about doing this, maybe we just keep it as-is until there's a compelling reason to duplicate `EnvVar` into this project.